travel-event-host-app/src/components/searchSection/searchEventsFilterBox/filterBox.tsx

```diff
@@ -100,7 +102,7 @@ const FilterBox = ({
               control={
                 <Checkbox checked={checked} onChange={handleCategoryChange} name={category} />
               }
-              label={category}
+              label={CategoryDict[category as Category]}
```

Comment:
Here was the source of confusion. The `name` property of the checkbox control can simply be the category enum (camelCase). 
The label is where we would want to use the dict (I think you called it CategoryArray - it actually isn't an array - it's an object, essentially dictionary). The dictionary will give us the friendly string we can use in the label